cli.js

```diff
@@ -39,10 +39,6 @@ gulp.on('error', function(err) {
     process.stderr.write(err);
 });
 
-gulp.on('failed', function(err) {
-    process.exit(err);
-});
```

Comment:
I'm guessing that the above removed code is no longer required if `gulp-runner` stays as it is, but would it make sense to leave it there in case the author reintroduces it.